SofaKernel/framework/sofa/core/ObjectFactory.cpp

```diff
@@ -112,14 +112,17 @@ objectmodel::BaseObject::SPtr ObjectFactory::createObject(objectmodel::BaseConte
     objectmodel::BaseObject::SPtr object = NULL;
```

Comment:
The modifs in the object factory need explanations to clearly get what the consequences are.